Formula/contentful-cli.rb

```diff
@@ -3,8 +3,6 @@
 class ContentfulCli < Formula
   desc "Contentful command-line tools"
   homepage "https://github.com/contentful/contentful-cli"
-  # 1.19.0 introduced a big code layout change, which is not easy to patch.
-  # TODO: re-add version throttling in next bump
```

Comment:
We probably want a comment like this here:

https://github.com/Homebrew/homebrew-core/blob/655e14ffe41969ddfd2541a7d9e2350b5824ec48/Formula/vim.rb#L4